app/src/main/res/xml/remote_config_defaults.xml

```diff
@@ -28,6 +28,11 @@
         <key>smsTimeoutSeconds</key>
         <value>900</value>
     </entry>
+<!--    Sms-auto-retrieval timeout supports only duration between 0-120 seconds -->
+    <entry>
+        <key>smsAutoRetrievalSeconds</key>
```

Comment:
Prohodil bych to - starý 120 sekund, nový 900 sekund. Kvůli backwards compatibility.

Reply:
Fixed